Packages/KeyAppKit/Sources/KeyAppNetwork/Encodable.swift

```diff
@@ -1,17 +1,25 @@
 //
-//  Codable+SnakeCase.swift
+//  Encodable.swift
 //  p2p_wallet
 //
 //  Created by Ivan on 28.04.2022.
 //
 
 import Foundation
 
-extension Encodable {
+public extension Encodable {
```

Comment:
It's not clear what this extension does

Reply:
This extension has been removed from KeyAppNetworking